AmplifyPlugins/Storage/AWSS3StoragePluginFunctionalTests/AWSS3StoragePluginTestBase.swift

```diff
@@ -102,4 +88,40 @@ class AWSS3StoragePluginTestBase: XCTestCase {
 
         return bucketValue
     }
+
+    func signUp() {
+        guard !AWSS3StoragePluginTestBase.isFirstUserSignedUp,
+              !AWSS3StoragePluginTestBase.isSecondUserSignedUp else {
+            return
+        }
+
+        let registerFirstUserComplete = expectation(description: "register completed")
+        let registerSecondUserComplete = expectation(description: "register completed")
+        AuthSignInHelper.signUpUser(
```

Comment:
We might need to update the README of storage auth section to add a pre-signup lambda to autoconfirm the user.